wiki/Tournaments/TSC/1/en.md

```diff
@@ -5,16 +5,16 @@ tags:
 ---
 # Taiko Suiji Cup
 
-![TSC Logo](logo.jpg)
+![TSC Logo](img/logo.png)
```

Comment:
About the jpg and png discussion on the template. It should be jpg if there is no transparency.

Reply:
in this case the png format works much better because of the large areas of solid color. with jpeg the only thing you'd be gaining is filesize ;)